tests/bricks/test_sequence_generators.py

```diff
@@ -100,6 +99,7 @@ def test_integer_sequence_generator():
     Such sequence generators can be used to e.g. model language.
 
     """
+    floatX = theano.config.floatX
```

Comment:
Basically because it was annoying and if you are modifying it during the
run of a single function you have bigger problems. I just wanted to get rid
of it at the module level: non persistent scopes are fine, selon moi.